### PR TITLE
Design new start page #1049

### PR DESCRIPTION
**Pull request by ksheremet**
Android:
Screenshot_1559738713
Screenshot_1559738752

Ios:
Screenshot_1559737414

### CODE REVIEW

**Comment by ksheremet** [comment]
@Kateevp 
What do you think about size, shadows, etc?
Is it better to leave "+" button as it is or make bottom transparent?

**Comment by Kateevp** [comment]
I suggest changing the size of the mask (extend it till the very bottom), move the button (+) to the right corner. Let me know if you need paddings.
Screenshot_2

**Comment by ksheremet** [comment]
Screenshot_1560074115
Screenshot_1560074181

File: flutter/lib/views/decks_list/deck_menu.dart [hunk 1]
```diff
@@ -16,8 +16,11 @@ const _animationDuration = Duration(milliseconds: 250);
 
 class DeckMenu extends StatefulWidget {
   final DeckModel deck;
+  final double size;
```
Comment (dotdoom) [inline flutter/lib/views/decks_list/deck_menu.dart]: `size` => `buttonSize`

File: flutter/lib/views/decks_list/decks_list.dart [hunk 2]
```diff
@@ -15,12 +15,12 @@ import 'package:delern_flutter/views/helpers/sign_in_widget.dart';
 import 'package:flutter/material.dart';
 import 'package:pedantic/pedantic.dart';
 
-class DecksList extends StatefulWidget {
-  final String title;
+const double _itemElevation = 4;
```
Comment (dotdoom) [inline flutter/lib/views/decks_list/decks_list.dart]: missing `k` in Konstant name

File: flutter/lib/views/decks_list/decks_list.dart [hunk 3]
```diff
@@ -120,100 +120,184 @@ class _DecksListState extends State<DecksList> {
...
-            // The size of FAB = 56 logical pixels from Material Design.
-            // To make settings available that are behind Fab, padding=60 was
-            // added.
-            const Padding(
-              padding: EdgeInsets.only(bottom: 60),
-            )
           ],
         ),
         floatingActionButton: CreateDeckWidget(key: fabKey),
+        //floatingActionButton: CreateDeckWidget(key: fabKey),
```
Comment (dotdoom) [inline flutter/lib/views/decks_list/decks_list.dart]: Unnecessary line?

File: flutter/ios/Podfile.lock [hunk 4]
```diff
@@ -232,4 +232,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: aff02bfeed411c636180d6812254b2daeea14d09
 
-COCOAPODS: 1.7.1
+COCOAPODS: 1.7.1
```
Comment (dotdoom) [inline flutter/ios/Podfile.lock]: I think build may not like this

File: flutter/lib/views/decks_list/decks_list.dart [hunk 5]
```diff
@@ -120,100 +120,184 @@ class _DecksListState extends State<DecksList> {
...
+  Widget build(BuildContext context) {
+    final emptyExpanded = Expanded(
+      flex: 1,
+      child: Container(
+        color: Colors.transparent,
+      ),
+    );
+
+    final primaryTextStyle = app_styles.primaryText.copyWith(
+        fontSize: height * 0.25 < app_styles.kMinPrimaryTextSize
```
Comment (dotdoom) [inline flutter/lib/views/decks_list/decks_list.dart]: Suggest using [`Math.max`](https://api.dartlang.org/stable/2.3.1/dart-math/max.html) instead

**Pull request merged**